test/node/to_bson_test.js

```diff
@@ -133,19 +131,24 @@ describe('toBSON', function () {
   });
 
   describe('when used on global existing types', () => {
-    beforeEach(() => {
+    beforeEach(function () {
+      if (__isWeb__) {
+        // These prototype modification do not make it
+        // to the BSON lib that was run in a separate context
+        return this.skip();
+      }
       Number.prototype.toBSON = () => 'hello';
       String.prototype.toBSON = () => 'hello';
       Boolean.prototype.toBSON = () => 'hello';
-      if (BigInt) BigInt.prototype.toBSON = () => 'hello';
+      BigInt.prototype.toBSON = () => 'hello';
```

Comment:
is the purpose of these extensions to just not throw? or is the output checked somewhere? if it *is* checked, then I think it would be better for the toBSON on the different types to return different strings, but otherwise it doesn't matter, though in that case, empty string or void would do just as well, no?

Reply:
It is being checked on line 157 inside `testToBSONFor` this is a bit of a table test without the loop, I was removing the if stmt here since we're testing on node versions that always have BigInt so it was never not testing the BigInt case. But I think that's a good enhancement to the test!